# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: b525ad6838f0
# Full Hash: b525ad6838f091c2ab7cfc52cda593eca7941fa8
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-05 22:07:34
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Add support for preloading key-only cursors. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44009
# ==============================================================================

diff -r c009bbdbbfd5 -r b525ad6838f0 dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 14:40:35 2019 +0000
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 14:40:31 2019 +0000
@@ -79,9 +79,9 @@
   void_t;
   nsresult;
   ObjectStoreCursorResponse[];
-  ObjectStoreKeyCursorResponse;
+  ObjectStoreKeyCursorResponse[];
   IndexCursorResponse[];
-  IndexKeyCursorResponse;
+  IndexKeyCursorResponse[];
 };
 
 protocol PBackgroundIDBCursor
